interface/usergroup/addrbook_edit.php

```diff
@@ -407,16 +412,16 @@ function typeSelect(a) {
  <tr>
   <td nowrap><b><?php echo xlt('Email'); ?>:</b></td>
   <td>
-   <input type='entry' size='40' name='form_email' maxlength='250'
-    value='<?php echo attr($row['email']); ?>'
+      <input type='entry' size='40' name='form_email' id = 'email_id' onblur="validateEmail()"  maxlength='250' 
```

Comment:
semicolon (please do not say semi-colon... it means something else!) separates style elements, but not required for the last attribute.  onblur should indeed end in a semicolon.